src/components/formio/radio.tsx

```diff
@@ -67,6 +75,22 @@ export const Radio: React.FC<RadioProps> = ({
       </div>
 
       {description && <Description text={description} />}
+
+      {value && isClearable && (
+        <div>
+          <button
+            type="button"
+            className="btn btn-secondary formio-button-remove-row"
+            onClick={() => setFieldValue(name, '')}
+          >
+            <i className={clsx('fa', 'fa-times-circle', 'offb-radio-default')} aria-hidden="true" />
+            <FormattedMessage
+              description="Clear selection button label"
+              defaultMessage="Clear selection"
+            />
```

Comment:
How does the spacing look if you just add some whitespace to the generated markup?

```suggestion
            <i className={clsx('fa', 'fa-times-circle', 'offb-radio-default')} aria-hidden="true" />
            {' '}
            <FormattedMessage
              description="Clear selection button label"
              defaultMessage="Clear selection"
            />
```
if this looks okay, then just remove the `offb-radio-default` class name and SCSS file.

Reply:
It works fine!I wouldn't have thought of this simple solution